core/config/docs/docs_test.go

```diff
@@ -92,6 +92,9 @@ func TestDoc(t *testing.T) {
 		docDefaults.Transactions.AutoPurge.Threshold = nil
 		docDefaults.Transactions.AutoPurge.MinAttempts = nil
 
+		// FinalityDepth has been updated to 10 in fallback and set to 50 in Ethereum_Mainnet
+		fallbackDefaults.FinalityDepth = docDefaults.FinalityDepth
```

Comment:
Not sure I understand why this needs to be done? are the tests failing without these?

Reply:
The test is setup in such a way that it wants the falback value to be reflected in the docDefaults, but since we updated the fallback and added a default for Ethereum chains this test fails without the new update.

Reply:
I still dont think we should be doing this, dont have the full context.
 _falback value to be reflected in the docDefaults_ - yes & we arent doing anything different, we want the fallback value to reflect in the defaults if it doesnt have one, how was it working all this time?